diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter2/HeaderCommentRuleFixTest.kt

```diff
@@ -71,6 +71,17 @@ class HeaderCommentRuleFixTest : FixTestBase(
         )
     }
 
+    @Test
+    @Tag(WRONG_COPYRIGHT_YEAR)
+    fun `should not raise npe`() {
```

Comment:
NPE occured when there was a file with copyright and year 2020 (like `2019-2020`) and it should have been fixed to `2019-2021`, but there was an NPE instead. You are checking a different case.

Reply:
In logs that you provided I saw NPE occurring on this case. I guess the problem is that we are checking only one file. As you said, we should check copyright in each file (`paragraph2/header`)

Reply:
We had `2020-2020` set in config for copyright text and I expected just to see failing tests: expected `2020-2021` and actual `2020-2020`. But there are NPEs all of a sudden. I guess that indicates some flaws in logic of checks.